src/tooltool/client/validate.sh

```diff
@@ -84,9 +81,10 @@ fi
 status "running tests (under coverage)"
 coverage erase || not_ok "coverage failed"
 coverage run --rcfile=coveragerc --source=tooltool `dirname \`which nosetests\``/.nosetests-wrapped test_tooltool.py || not_ok "tests failed"
+#coverage run --rcfile=coveragerc --source=tooltool `which nosetests` test_tooltool.py || not_ok "tests failed"
```

Comment:
Blocker: I think we should bring coverage back.

Reply:
one line above is running coverage. below line was added since using tox and nix is a bit different. once we migrate to non-nix repo i will remove the nix specific line and uncomment the line you are pointing to.

Reply:
Okay. In this case, could you remove this line or put a code comment? Leaving the line this way may confuse future readers (including ourselves).

Reply:
done